CHANGELOG.md

```diff
@@ -20,7 +20,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
```

Comment:
I don't think we should change the name in the changelog since this is a historical record.  I would expect that there would be a line describing this rename in the changelog. e.g. rename project to SIL.Windows.Forms.Archiving